region/brinstar/kraid/Baby Kraid Room.json

```diff
@@ -649,9 +649,13 @@
       },
       "devNote": [
         "Max extra run speed $7.0.",
-        "Using the full runway, there is a 2-frame window for the jump,",
-        "then a 1-frame or 2-frame window for the morph depending on the jump (with a last-frame jump giving a 2-frame morph window).",
-        "If there is a solid tile on the ceiling in the next room past the door frame, then a the jump and morph are both frame-perfect."
+        "Avoid backing into the corner at the start of runway;",
+        "instead press against it and turn around, to put Samus into a better position.",
+        "It can help to perform to advance 2 more pixels by performing an arm pump: press and release an angle button one time while running.",
```

Comment:
```suggestion
        "It can help to advance 2 more pixels by performing an arm pump: press and release an angle button one time while running.",
```